x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/SqlTranslateRequest.java

```diff
@@ -64,8 +64,12 @@ public static SqlTranslateRequest fromXContent(XContentParser parser) {
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         // This is needed just to test parsing of SqlTranslateRequest, so we can reuse SqlQuerySerialization
-        return new SqlQueryRequest(query(), params(), zoneId(), fetchSize(), requestTimeout(),
-            pageTimeout(), filter(), null, null, requestInfo()).toXContent(builder, params);
+        return new SqlQueryRequest(query(), params(), zoneId(), fetchSize(), requestTimeout(), pageTimeout(), 
+            filter(), 
+            null, 
+            null, 
+            requestInfo(),
+            false).toXContent(builder, params);
```

Comment:
Instead of `false` value for multi_value_leniency here why not using `Protocol.FIELD_MULTI_VALUE_LENIENCY `?